src/components/presale/presale.tsx

```diff
@@ -277,6 +294,28 @@ class Presale extends Component<PRESALEPROPS, PRESALESTATE> {
     return StoreClasses.store._getPresaleContractAddress() || '';
   }
 
+  _calculateWOLF(val: string | undefined): string {
+    return val && this.state.inputValid
+      ? (parseFloat(val) * 40).toFixed(2).toString()
```

Comment:
This calculation seems wrong,

![wrong-wolf](https://user-images.githubusercontent.com/15795328/104857006-7258ea80-58ca-11eb-8f25-94f32306eafc.png)

Reply:
thx, as discussed: ETH is now darker and better readable, we are currently at 40 WOLF / ETH so calculation is correct.